From a0babcf8e926f674e856cb62af42031305b77f87 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 29 Nov 2024 19:45:40 +0100
Subject: [PATCH 1521/1695] plat-stm32mp1: select
 CFG_STM32MP1_PM_CONTEXT_VERSION = 4

Change the default used stm32mp1 PM mailbox context to version 4.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1741d18b5adc51ad767d6813a7c018243437a088 plat-stm32mp1: select CFG_STM32MP1_PM_CONTEXT_VERSION = 4]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/423283
Change-Id: I41e8210d7fe2bac42cf5685145894d96209e2f2c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/437079
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 6435b8b74..69536d317 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -432,8 +432,8 @@ CFG_STM32_TAMP_NVRAM ?= y
 # Default enable PKCS11 TA to have PKCS11 tests built in OP-TEE Test
 CFG_PKCS11_TA ?= y
 
-# Default use stm32mp1 PM mailbox context version 3
-CFG_STM32MP1_PM_CONTEXT_VERSION ?= 3
+# Default use stm32mp1 PM mailbox context version 4
+CFG_STM32MP1_PM_CONTEXT_VERSION ?= 4
 
 CFG_HWRNG_PTA ?= $(CFG_STM32_RNG)
 ifeq ($(CFG_HWRNG_PTA),y)
-- 
2.47.2

